src/services/netMgr.js

```diff
@@ -203,17 +222,31 @@ NetMgr.setLocalStorageFromToken = function(token) {
 
 // Add interceptor to detect an expired access_token and refresh;
 NetMgr.axiosInstance.interceptors.response.use(
-    function (response) {
+    (response) => {
+        // If the request was successfully completed set the online status to true.
+        NetMgr.setOnlineStatus(true);
+
         // everything fine! return the response to apiGet/apiPost
         return response;
     },
-    function (error) {
+    (error) => {
         // Get the original request configuration.
         var origCfg = error.config;
 
         // Get the error from that request.
         var origResp = error.response;
```

Comment:
This has a more verbose version of error.message so I used error.message instead. I want to find a better way of checking of we're offline than just "Network Error" but this will do fine for now.